src/seedu/addressbook/parser/Parser.java

```diff
@@ -101,7 +101,7 @@ public Command parseCommand(String userInput) {
      * @param args full command args string
      * @return the prepared command
      */
-    private Command prepareAdd(String args) {
+    private static Command prepareAdd(String args) {
```

Comment:
based on the files changed, I am guessing, you have not edited that part of the code where you create a parser object.
after making this method static, you can remove the parser object from your code!